scripts/dts/python-devicetree/src/devicetree/edtlib.py

```diff
@@ -1824,7 +1824,8 @@ def __repr__(self):
             compat = f" for compatible '{self.compatible}'"
         else:
             compat = ""
-        return f"<Binding {os.path.basename(self.path)}" + compat + ">"
+        basename = os.path.basename(self.path) if self.path is not None else ""
```

Comment:
```suggestion
        basename = os.path.basename(self.path or "")
```

Reply:
Thanks for the review.

I've applied your change (more pythonesque, indeed), but not by clicking "Commit suggestion" to avoid uselessly triggering the CI tests twice: once when I commit the suggestion, then once again after the rebase/squash/force push.

Is it fine if I don't forget to add the "Co-authored-by:" tag ?

Reply:
we rarely use the Co-authored-by, unless suggestions are a significant part of the implementation. But it's ok to use it.

Reply:
Point taken, makes more sense to me too. So skipping the "Commit suggestion" button should be fine, provided I let the reviewer know that the change is applied.